packages/loader/driver-utils/src/networkUtils.ts

```diff
@@ -3,7 +3,8 @@
  * Licensed under the MIT License.
  */
 
-import { type ITelemetryErrorEventExt, ITelemetryLoggerExt } from "@fluidframework/telemetry-utils";
+import { type ITelemetryErrorEventExt } from "@fluidframework/telemetry-utils/internal";
```

Comment:
Nit: Since the only import from this module is a type import, we can improve this a bit by changing it to:
```suggestion
import type { ITelemetryErrorEventExt } from "@fluidframework/telemetry-utils/internal";
```

This has the benefit avoiding importing the module at all at runtime.

Reply:
There's actual functional differences between the two 😮 ? TIL.

Reply:
That's my understanding, yes. TypeScript can't necessarily tell whether or not a non-type-only import statement is needed for side-effects. This is a way to explicitly opt out of the import.